tf/train.py

```diff
@@ -32,33 +32,20 @@
 SKIP_MULTIPLE = 1024
 
 
-def get_chunks(data_prefix):
-    return glob.glob(data_prefix + "*.gz")
-
-
 def get_all_chunks(path):
     chunks = []
-    for d in glob.glob(path):
-        chunks += get_chunks(d)
+    for dir in glob.glob(path):
+        chunks += glob.glob(f"{dir}*.gz")
     return chunks
 
 
 def get_latest_chunks(path, num_chunks, allow_less):
     chunks = get_all_chunks(path)
-    if len(chunks) < num_chunks:
-        if allow_less:
-            print("sorting {} chunks...".format(len(chunks)), end='')
-            chunks.sort(key=os.path.getmtime, reverse=True)
```

Comment:
this section appears again on line 62, no need to run it twice. the only difference is the line `chunks = chunks[:num_chunks]` that appears on line 64 but not in this section, but it's fine for it to run later, it just takes a slice of the whole array